Guidelet/GuideletLoadable.py

```diff
@@ -190,7 +190,8 @@ def addValuesToDefaultConfiguration(self):
                    'RecordingFilenamePrefix' : 'GuideletRecording-',
                    'RecordingFilenameExtension' : '.mhd',
                    'SavedScenesDirectory' : defaultSavePath,
-                   'UltrasoundBrightnessControl' : 'Buttons'
+                   'UltrasoundBrightnessControl' : 'Buttons',
+                   'RecordingAlwaysEnabled' : 'False',
```

Comment:
"Always" is too general. We may still want to make it disabled in certain cases. Change it to: "RecordingEnabledWhenConnectorNodeDisconnected".